laravel5/app/Monomalpoly/Joueur.php

```diff
@@ -0,0 +1,54 @@
+<?php
+class Joueur{
+	private $id;
+	private $nom;
+	private $couleur;
+	private $posX;
+	private $posY;
+	private $nbDisque;
+	private $listeServeur;
+
+	function Joueur($id,$nom,$couleur){
+		$this->posX = 0;
+		$this->posY = 0;
+		$this->nbDisque = 0;
+		$this->listeServeur = [];
+
+		$this->id = $id;
+		$this->nom = $nom;
+		$this->couleur = $couleur;
+		
+	}
+
+	function move(){
+		$firstDice = random_int(1,6);
+		$secondDice = random_int(1,6);
+		$moveNb = $firstDice + $secondDice;
+		$canMove = testPosition($moveNb);
+		if($canMove == true){
```

Comment:
Same as `if($canMove){`

Reply:
Rather than testing if it is normal and then doing the move, you better should test before actually doing the move.

Exemple:
```
if(!$canMove){
    throw new Exception("Some message");
}
```

This way, you forget handling edge cases much less often.